java-rx-smgrsdk/src/main/java/io/vantiq/svcconnector/SvcConnSvrMessage.java

```diff
@@ -9,11 +11,13 @@
  * <p>
  * All rights reserved.
  */
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Adding the new property to server requests breaks existing storage managers, since it tries to shove it into this class but can't find it. This JsonIgnore will let future properties be quietly ignored if this class doesn't have it.

The Python connectors don't have this issue.